android/app/src/debug/java/com/reactnativesampleapp/ReactNativeFlipper.java

```diff
@@ -4,7 +4,7 @@
  * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
  * directory of this source tree.
  */
-package com.reactnativesampleapp;
+package com.BitriseReactNativeSample;
```

Comment:
I would be inclined to keep the package name all lowercase; that's the Java standard.

Reply:
Good catch! I'll update that.

- [ ] Lowercase the package name in the Android app

Reply:
Updated in https://github.com/bitrise-io/getting-started-react-native-sample-app/pull/3/commits/6b3fd59c95779d431efb41568e3694da22f0e72f